mkdocs.yml

```diff
@@ -168,6 +168,8 @@ pages:
       - Solr: integration/solr-integration.md
       - Varnish Cache: integration/varnishcache-integration.md
       - Windows: integration/windows.md
+        - Configuring Logs Collection: integration/configuring-windows-agent.md
```

Comment:
Please also update this doc with the new CMD/PowerShell commands: https://github.com/sematext/docs/blob/master/docs/agents/sematext-agent/windows-installation.md